bubblewrap.c

```diff
@@ -926,6 +934,25 @@ parse_args (int *argcp,
           argv += 1;
           argc -= 1;
         }
+      else if (strcmp (arg, "--exec-label") == 0)
+        {
+          if (argc < 2)
+            die ("--exec-label takes an argument");
+
+          opt_exec_label = argv[1];
+          argv += 1;
+          argc -= 1;
+        }
+      else if (strcmp (arg, "--file-label") == 0)
+        {
+          if (argc < 2)
+            die ("--file-label takes an argument");
+
+          opt_file_label = argv[1];
+          label_create_file(opt_file_label);
```

Comment:
I wanted to avoid putting SELinuxisms into bubblewrap, since I thought others (SMACK) might want to add support.  So I went with generic We can add an label_support() function and error out, if there is no support.